src/config/section/image.js

```diff
@@ -94,9 +102,14 @@ export default {
           args: (record, store) => {
             var fields = ['name', 'displaytext', 'passwordenabled', 'ostypeid', 'isdynamicallyscalable']
             if (['Admin'].includes(store.userInfo.roletype)) {
-              fields.push('isrouting')
+              fields.push('isrouting', 'templatetype')
             }
             return fields
+          },
+          mapping: {
+            templatetype: {
+              options: ['BUILTIN', 'USER', 'SYSTEM', 'ROUTING', 'PERHOST']
+            }
           }
         },
```

Comment:
What is PERHOST @ravening @weizhouapache ?

Reply:
@rhtyd removed `PERHOST` as its not needed